cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -546,6 +560,243 @@ func (uc *UpstreamController) updateNodeStatus() {
 	}
 }
 
+// createEdgeCluster create a new edgeCluster
```

Comment:
the function name is pretty good already. this comment doesn't seem needed.

Reply:
deleted.